### DISCUSSION TITLE
Why does a user need to be a staff user to create projects on Doccano?

### DISCUSSION THREAD

**Discussion by Vydruth** [doccano/doccano#1751] [General]
Hi @Hironsan and @youichiro 

Regarding: https://github.com/doccano/doccano/commit/5f161ce2418920f022c816cf3eca2f624beb291b

Why does a user need to be a staff user to create a new project? It kinda restricts the users from using the tool to create their own annotation tasks and share with other users. In a organizational setting Staff Users are mostly just a select number of people.

So would like to understand this change and make sure we are purposefully heading in the direction keeping all risks in mind.

This change will also cause issues when Okta implementation is brought back, as not all users who login with Okta are staff users. They need to be marked as staff users explicitly. Meaning they have additional permissions to other tools that Okta provides. (Unless I am mistaking how this works.)  Please let me know your thoughts. My team would personally vote to revert this change but happy to hear the other side of the story.

Thank you!

**Comment by Hironsan** [comment]
I assumed that the no-staff users are cloud workers. After adding them to the doccano, we don't want them to create a project. That's why creating a new project is restricted to the staff users. But as you know, I think it is too restrictive.

I wrote some idea to solve the problem in the past. 
- https://github.com/doccano/doccano/issues/935

Welcome to your opinion.

**Reply by Vydruth** [reply]
Hi Hiron, sorry for the delay in responding. My team discussed this change internally and **we are aligned with your idea** around user privileges. Thanks for sharing the documentation. We use Okta for Authenticating and Authorizing our users. The issue we are seeing today while upgrading to 1.5.5 is that by default all users created on Okta are synced as non-staff users in the Django app.
 
We tried to use Django Admin Console to update the user to a staff user but as soon as the user performs a login, the staff user permission gets removed automatically from Django app(Maybe the code pulls user privs from Okta, removing the staff user setting on Django). This is something we can look into and have a PR fixing the issue as soon as we identify the root cause. 

Thank you again for all the hard work you put in for this project.
